server/controllers/notification.js

```diff
@@ -0,0 +1,24 @@
+const Notification = require("../models/Notification");
+const User = require("../models/User");
+const asyncHandler = require("express-async-handler");
+
+exports.notifications = asyncHandler(async (req, res, next) => {
+  const notification = await Notification.findById({ userId: req.user.id });
```

Comment:
`findById` will return only a single notification, but users will have more than one notification in most cases.